karma.conf.js

```diff
@@ -36,7 +37,6 @@ module.exports = (config) => {
               presets: ['airbnb'],
             },
           },
-          { test: /\.json$/, loader: 'json-loader' },
```

Comment:
why is this removed?

Reply:
This was throwing an error when running karma and is not needed with webpack 4.